sound/soc/sof/pm.c

```diff
@@ -80,11 +80,18 @@ static int sof_resume(struct device *dev, bool runtime_resume)
 
 	/*
 	 * Nothing further to be done for platforms that support the low power
-	 * D0 substate.
+	 * D0 substate, and we need to re-enable the DMA trace if it was stopped
```

Comment:
CHange to comments to 'Re-enabled trace and return when resuming from low-power D0 substate"